examples/connectivity/plot_canica_resting_state.py

```diff
@@ -26,6 +26,9 @@
 adhd_dataset = datasets.fetch_adhd()
 func_filenames = adhd_dataset.func  # list of 4D nifti files for each subject
 
+# print basic information on the dataset
+print('Functional nifti images are at: %s' % adhd_dataset.func[0])  # 4D data
```

Comment:
I would use "First functional nifti image is at: " otherwise this is a bit misleading.

Other examples would need to be harmonised in a similar fashion.